opm/common/OpmLog/Logger.cpp

```diff
@@ -63,6 +63,11 @@ namespace Opm {
             return true;
     }
 
+    void Logger::clearBackends() {
```

Comment:
Can I suggest `drop / delete / remove / ...` the name `clear()` seems to indicate that only the internal state of the backends is cleared/reset.